dev-utilities/committer-utils/prepare-commit

```diff
@@ -16,242 +16,31 @@
 #  limitations under the License.
 #
 
-# not likely to change
-METRON_UPSTREAM="https://git-wip-us.apache.org/repos/asf/metron.git"
-BRO_PLUGIN_UPSTREAM="https://git-wip-us.apache.org/repos/asf/metron-bro-plugin-kafka.git"
-CONFIG_FILE=~/.metron-prepare-commit
-GITHUB_REMOTE="origin"
-BASE_BRANCH=master
+LOCATION_DIR=`dirname $0`
+#shellcheck source=./metron-committer-common
```

Comment:
Does this need to be commented out?

Reply:
Yes, this a directive for shellcheck

https://github.com/koalaman/shellcheck/wiki/SC1090
https://github.com/koalaman/shellcheck/wiki/Directive